client/domain/FeedbackObjectFactory.js

```diff
@@ -22,6 +22,10 @@ tie.factory('FeedbackObjectFactory', [
     var Feedback = function(answerIsCorrect) {
       this._paragraphs = [];
       this._answerIsCorrect = answerIsCorrect;
+      // This records what message was displayed with this feedback.
+      // If no message (or no message with additional potential messages)
+      // was displayed, this will remain at -1.
```

Comment:
It lets me +1 it to 0, which is handy instead of having to have it as a separate case. The name suggests an int to me, anyways.